### DISCUSSION TITLE
Brew brings non-free version of master-pdf-editor

### DISCUSSION THREAD

**Discussion by kirk86** [Homebrew/discussions#773] [Casks]
Is there any possibility to modify the current cask or add an additional cask for the free version without the watermark?
The current version `brew install master-pdf-editor` installs the version which automatically adds a watermark to any pdf upon save!

**Comment by SMillerDev** [comment]
There is no free version without the watermark as [the website explains](https://code-industry.net/masterpdfeditor/):
> The demo version allows you to try out all the features of Master PDF Editor. There are no limitations except for the addition of a watermark on the output file.

**Reply by kirk86** [reply]
In the linux world there's an[ older version without watermark](https://aur.archlinux.org/packages/masterpdfeditor-free/) can't we use that or provide a cask for that as well?

**Reply by SMillerDev** [reply]
The current version is the latest version that is available upstream. We're not going to keep unsupported versions of software around so people don't need to pay for features.

**Reply by kirk86** [reply]
No need for hostility, behind a keyboard everyone is the next level G, in addition, the contrapositive of your statement means that you're going to keep unsupported versions of software as long as people pay for features.

**Reply by SMillerDev** [reply]
Not really, we'll keep unsupported versions around if other software depends on it. No other reasons, including the one I mentioned, are a reason to keep unsupported software around.
Twisting my words won't magically make support happen.

**Reply by kirk86** [reply]
> Twisting my words

Look at yourself, don't blame others.

> won't magically make support happen.

I don't give a shite!

**Reply by carlocab** [reply]
Please adhere to the [code of conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct) when posting here.
